src/Estimators/MomentumDistribution.h

```diff
@@ -77,13 +77,15 @@ class MomentumDistribution : public OperatorEstBase
 
   //MomentumDistribution(const MomentumDistribution& md);
```

Comment:
is it better if put this one = default and under private to prevent unintended use from outside?

Reply:
Probably